js/map.js

```diff
@@ -216,6 +216,15 @@ function showLayer(selectedGeography, selectedCategory, selectedStatus) {
   updateLegend(selectedGeography, selectedCategory, selectedStatus)
 }
 
+// This function takes in a button as an element, sets the class 'active' to the element
+// adds the aria-pressed value as "true" to the element, then returns the element
+function toggleActive(e){
```

Comment:
i think we could put most of our logic in this function and have it set both the `class` and `aria-pressed` values to `true`/ `false` based on the current attribute value. you could then just call this function in the 3 places where we have our button listeners.